migrations/sql/V1.14__Add_party_appt_and_party_appt_type.sql

```diff
@@ -1,35 +1,53 @@
-CREATE TABLE public.mine_party_appt_type_code (
-    mine_party_appt_type_code varchar(3) NOT NULL,
-    description varchar(100) NOT NULL,
-    display_order int4 NULL,
-    active_ind bool NOT NULL DEFAULT 'yes',
-    create_user varchar(60) NOT NULL,
-    create_timestamp timestamptz NOT NULL DEFAULT now(),
-    update_user varchar(60) NOT NULL,
-    update_timestamp timestamptz NOT NULL DEFAULT now(),
-    CONSTRAINT party_mine_xref_type_code_pk PRIMARY KEY (mine_party_appt_type_code)
+CREATE TABLE public.mine_party_appt_type_code
```

Comment:
@NWCalvank @FreshGyan Not sure why this migration file is in Jons PR, thinking he just saved it and it autoformatted. 

I wanted to check in with you to make sure this doesn't break anything if we merge it in.

Reply:
This will break in the pipeline, yes. It's important that everyone on the team is careful about only committing files that should be and were intentionally changed.

This is the change that broke the builds in the pipeline yesterday.

Reply:
So assuming that this is just indentation changes, is it okay to be merged in? 

He didn't change this file, just rebased with develop. 
 I'm not sure what steps to take next.

Reply:
Nope, no indentation changes either. The file checksum changes even if you add a single space.
This file needs to not be a part of this PR